src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
```

Comment:
Similarly, replace the hard-coded value `3` with a constant to explain what this value represents (e.g., `SHORT_TERM_RENTAL`). Also, replace `20` with a constant (e.g., `SHORT_TERM_DISCOUNT`).